Gemfile

```diff
@@ -3,10 +3,12 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 ruby '3.0.0'
 
+gem 'pg'
```

Comment:
It's a good practice to specify the version range that you want to use for the gems.